app/workers/functions.php

```diff
@@ -29,7 +29,8 @@
 
 $dockerUser = App::getEnv('DOCKERHUB_PULL_USERNAME', null);
 $dockerPass = App::getEnv('DOCKERHUB_PULL_PASSWORD', null);
-$orchestration = new Orchestration(new DockerAPI($dockerUser, $dockerPass));
+$dockerEmail = App::getEnv('DOCKERHUB_PULL_EMAIL', null);
```

Comment:
If we're adding this var, it should be in the variables config as well as the changelog (Not sure where else)
ref: https://github.com/appwrite/appwrite/blob/master/app/config/variables.php